sql/moz-fx-data-shared-prod/accounts_backend_external/nonprod_accounts_v1/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+---
+friendly_name: FxA Accounts nonprod
+description: >
+  A mirror of the `accounts` table from the staging FxA CloudSQL database,
+  updated daily to match the current state of the table.
+owners:
+  - akomar@mozilla.com
+labels:
+  application: accounts_backend
+  schedule: daily
```

Comment:
I've sometimes specified `incremental: false` under `labels:` for non-incremental ETLs. Labels with `false` values are ignored when deploying the table, so it isn't visible in BigQuery itself, but I figure maybe it's a helpful indication for us developers. 🤷